app/controllers/report_controller/reports/editor.rb

```diff
@@ -332,31 +332,31 @@ def reset_report_col_fields
     @edit[:miq_exp]               = true
   end
 
+  TAB_TITLES = {
+    'edit_1' => _('Columns'),
```

Comment:
I'm afraid `_()` inside a frozen constant will not work. You have to use `N_()` here and `_()` later when you actually use the constant.